packages/docusaurus-theme-common/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Common code for Docusaurus themes.",
   "main": "./lib/index.js",
   "types": "./lib/index.d.ts",
+  "sideEffects": [
+    "lib/components/Details/*"
```

Comment:
isn't something like "*.css" enough here?

Reply:
No, because that would still make the CSS lazy-loaded and mess with insertion order. You can check the commit history for my struggle.